site/content/docs/main/config/jwt-verification.md

```diff
@@ -78,6 +77,13 @@ spec:
 In the above example, the default route requires requests to carry JWTs that can be verified using provider-1.
 The second route _excludes_ requests to paths starting with `/css` from JWT verification, because it does not have a JWT verification policy.
 
+### Configuring TLS validation for the JWKS server
```

Comment:
Copied this content mostly verbatim from the same subject on the upstream-tls.md page.